managed/services/telemetry/config.go

```diff
@@ -183,9 +187,41 @@ func (c *ServiceConfig) Init(l *logrus.Entry) error { //nolint:gocognit
 		}
 	}
 
+	testClickhouseDSN := c.testClickhouseDSN()
+	if testClickhouseDSN != "" {
+		c.l.Debugf("Overriding Telemetry.DataSources.QanDBSelect.DSN with environment variables [%s][%s][%s][%s] to %q.", envClickhouseAddr, envClickHouseDatabase, envClickhouseBlockSize, envClickhousePoolSize, testClickhouseDSN)
+		c.DataSources.QanDBSelect.DSN = testClickhouseDSN
+	}
+
 	return nil
 }
 
+func (c *ServiceConfig) testClickhouseDSN() string {
```

Comment:
why `testClickhouseDSN`? why it is `test`?

Reply:
because in documentation is PMM_TEST_*

Reply:
PERCONA_TEST_ means that we are testing this environment variable yet, but it will be renamed on GA. So I agree with @ritbl